assembly/bindgen.ts

```diff
@@ -129,8 +135,8 @@ function encode<T, Output = Uint8Array>(
         if (value instanceof u128) {
           encoder.setString(name, value.toString());
         } else if (value instanceof Map) {
-          //@ts-ignore
           assert(
+            //@ts-ignore
```

Comment:
can we ignore a specific error instead of everything?

Reply:
Last time I sought this out I couldn't find out how.  It's not like tslint, which let's you ignore specific errors.  I did find an issue where someone asked about it, but I don't think it went anywhere.  This is the type of error that TS probably doesn't want to hind.